plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/scripts/lsp-server.sh

```diff
@@ -0,0 +1,8 @@
+rm -rf ca-lsp-server.tar
+rm -rf ca-lsp-server
+
+wget https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/releases/download/0.0.5/ca-lsp-server.tar
+
+archive=ca-lsp-server.tar
+mkdir ${archive%.tar*} 
+tar --extract --file=${archive} --strip-components=1 --directory=${archive%.tar*}
```

Comment:
You could probably do all this in a single line if you pile the output of a curl directly into tar, but if this works, it's fine.

Reply:
@nickboldt, It does uses [1] here.
[1] [1] https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/All/job/jbosstools-fabric8analytics-lsp-server_master/

Is there another way to point to the upstream CI build?